src/librustdoc/html/render/context.rs

```diff
@@ -73,7 +75,7 @@ crate struct Context<'tcx> {
 
 // `Context` is cloned a lot, so we don't want the size to grow unexpectedly.
 #[cfg(all(target_arch = "x86_64", target_pointer_width = "64"))]
-rustc_data_structures::static_assert_size!(Context<'_>, 144);
+rustc_data_structures::static_assert_size!(Context<'_>, 184);
```

Comment:
This increase seems to be fine, as when cloned, we create a new map, so perf is neutral (#91257)